client/src/app/core/repositories/agenda/agenda-item-repository.service.ts

```diff
@@ -178,4 +180,24 @@ export class AgendaItemRepositoryService extends BaseRepositoryWithActiveMeeting
         });
         return duration;
     }
+
+    public bulkOpenItems(items: ViewAgendaItem[]): Promise<void> {
+        const payload: AgendaItemAction.UpdatePayload[] = items.map(item => ({ closed: false, id: item.id }));
+        return this.sendBulkActionToBackend(AgendaItemAction.UPDATE, payload);
+    }
+
+    public bulkCloseItems(items: ViewAgendaItem[]): Promise<void> {
+        const payload: AgendaItemAction.UpdatePayload[] = items.map(item => ({ closed: true, id: item.id }));
+        return this.sendBulkActionToBackend(AgendaItemAction.UPDATE, payload);
+    }
+
+    public bulkSetAgendaType(items: ViewAgendaItem[], agendaType: AgendaItemType): Promise<void> {
+        const payload: AgendaItemAction.UpdatePayload[] = items.map(item => ({ id: item.id, type: agendaType }));
+        return this.sendBulkActionToBackend(AgendaItemAction.UPDATE, payload);
+    }
+
+    public bulkRemoveItemsFromAgenda(items: ViewAgendaItem[]): Promise<void> {
+        const payload: AgendaItemAction.DeletePayload[] = items.map(item => ({ id: item.id }));
```

Comment:
Just a notice. YOu often did not include the `DeletePayload` since it is mostly `Identifiable`. Maybe this is a good reason to include it. If you have time, please add the `DeleteAction` to each model.